library/pk_wrap.c

```diff
@@ -447,24 +447,24 @@ static int rsa_check_pair_wrap(mbedtls_pk_context *pub, mbedtls_pk_context *prv,
 {
     (void) f_rng;
     (void) p_rng;
-    return mbedtls_rsa_check_pub_priv((const mbedtls_rsa_context *) pub->pk_ctx,
-                                      (const mbedtls_rsa_context *) prv->pk_ctx);
+    return mbedtls_rsa_check_pub_priv((const void *) pub->pk_ctx,
+                                      (const void *) prv->pk_ctx);
```

Comment:
What is the benefit of changing this cast?

Reply:
Such casts are redundant and less maintainable. C standard is well defined with casts between `void*` and other types.

Reply:
The casts that were in place make it clear what is happening. Since `void *` is a universal donor and universal recipient, it removes the little type safety that there previously was. Please remove this change, and the other similar one. If you think they are absolutely necessary, you can then raise another PR with those in it, and the discussion can continue there, but otherwise the useful changes in here will get stalled while we argue.

(This is why multiple different types of changes in a single PR are an anti-pattern)